src/lib/drivers/extension.js

```diff
@@ -145,7 +145,7 @@ class ExtensionDriver extends Driver {
     return new Promise((resolve, reject) => {
       chrome.tabs.query(currentTab, tabs => {
         if (chrome.runtime.lastError) {
-          return reject(chrome.runtime.lastError);
+          return reject(chrome.runtime.lastError.message);
```

Comment:
You'll need to update the extension, too: https://github.com/GoogleChrome/lighthouse/blob/master/extension/app/src/popup.js#L57

It expects to be able to pull the `message` property.